.github/workflows/pull_request_examples.yml

```diff
@@ -120,7 +119,7 @@ jobs:
                   cp '${{ github.workspace }}/examples/${{ matrix.example }}/.env.example' .env
                   sed -i 's|NEXT_PUBLIC_WP_HOME_URL=.*|NEXT_PUBLIC_WP_HOME_URL=http://localhost|' .env
                   npm i '${{ github.workspace }}/packages/cli' --no-save
```

Comment:
(assuming @ayushnirwal approves these deviations):

We need comments here and anywhere else we're deviating from `DEVELOPMENT.md`, i.e. explaining why we're using `npm install` + `npx snapwp` instead of `npm link` + `snapwp`